applovin_max/example/lib/main.dart

```diff
@@ -73,6 +73,8 @@ class _MyAppState extends State<MyApp> {
       logStatus("SDK Initialized: ${configuration.countryCode}");
 
       attachAdListeners();
+
+      preloadAdViewAd();
```

Comment:
nit, but I feel we should add a comment and/or update the method name to indicate what we are demonstrating here, since this isn't typical integration technique.

Like, `// If you need to preload banners/MRECs ahead of time such that the contents are readily available when displayed` ...